pkg/ring/kv/dynamodb/metrics.go

```diff
@@ -33,9 +34,15 @@ func newDynamoDbMetrics(registerer prometheus.Registerer) *dynamodbMetrics {
 		Help: "Total consumed capacity on dynamodb",
 	}, []string{"operation"})
 
+	dynamodbCasAttempts := promauto.With(registerer).NewCounter(prometheus.CounterOpts{
+		Name: "dynamodb_kv_cas_attempt_total",
+		Help: "DynamoDB KV Store Attempted CAS operations",
+	})
```

Comment:
I am curious about this new metric. It is a counter so it is used to tell the rate of CAS attempts to DDB.
What's the usecase of having this rate metric?
If we just want to know the retries we got from this change, it might be hard to tell from the metrics itself

Reply:
Yeah, I saw there is similar metric in memberlist. Not sure how useful it will be. I agree that it will not be able to catch the retries in this change.